docs/governance/paper-lifecycle.md

```diff
@@ -0,0 +1,84 @@
+# Publishing Papers in the OpenWallet Foundation Community
+
+## Overview
+
+This document outlines the process for publishing papers in the OpenWallet Foundation technical community. These papers will be created as deliverables by either [task forces](./task-force-process.md) or [special interest groups](./special-interest-group-process.md).
+
+## Process
+
+There are three stages of an OWF paper -- draft, review, and approved.
+
+```mermaid
+flowchart
+  d[Draft]
+  r[Review]
+  t{TAC\nDecision}
+  a[Approved]
+
+  d -->|Submission| r
+  r --> t
+  t -->|Request Changes| d
+  t -->|Approve| a
+```
+
+A paper begins as a "draft" and retains this status until submitted to the TAC for review. At this point, the paper will be considered to be in the "review" stage and retains this status until the TAC makes a decision. The TAC may reject the paper for various reasons. If the paper is rejected by the TAC then it will return to a "draft" stage where the community can address the concerns of the TAC. The paper may also be accepted by the TAC, in which case it will be published as an "approved" paper of the OWF technical community. Only after the paper is accepted, may the paper use the "OpenWallet Foundation" trademark. See [required contents](#required-contents) for more information on what should be included within an "approved" paper.
+
+### Draft
+
+During the "draft" stage, the paper is being prepared by the task force or special interest group. This includes writing the technical content and ensuring that it meets the standards of the OWF technical community. If there are any concerns about the technical content that cannot be addressed by consensus of the task force or special interest group members, then the community may address these concerns during the "review" stage.
+
+### Submission
+
+The task force or special interest group lead may submit a version of their paper to the Technical Advisory Committee (TAC) for review. They can do this by sending an email to the [TAC mailing list](mailto:tac@lists.openwallet.foundation). The email should include the following information:
+
+- A brief description of the paper.
+- A link to the paper. Ideally, this should be a link to a pull request that contains the version of the paper where people in the community and the TAC voting members will be able to provide their feedback.
+- A link to the task force or special interest group that submitted the paper.
+
+### Review
+
+The applicable period to review a submitted paper will be no shorter than four weeks. The TAC will make reasonable efforts to provide feedback on the submitted version during the review period and provide any critical comments or objections, with sufficient specificity for the task force or special interest group members to respond and, if required, to facilitate resolution. 
```

Comment:
I would welcome a broader review process open to the whole foundation.

Reply:
I have attempted to make this clearer in the information note following this paragraph.

> Comments and feedback can be provided by anyone in the OWF community. The TAC will take into consideration any comments and feedback provided during the review process when making their decision.

Reply:
thanks!